content/concepts/target-market-settings.md

```diff
@@ -0,0 +1,179 @@
+# Target market settings
+
+Product information enriched in the PIM is meant to be distributed over your channels. Otherwise gathering so many information into one single source of truth would be quite useless, right? :wink:
```

Comment:
```suggestion
Product information enriched in the PIM is meant to be distributed over your channels. Otherwise gathering so much information into one single source of truth would be quite useless, right? :wink:
```